src/test/java/com/parasoft/demoapp/graphql/ItemGraphQLDataFetcherTest.java

```diff
@@ -108,4 +112,169 @@ public void test_getItems_incorrectAuthentication() throws IOException {
                 .and()
                 .assertThatField(ITEM_DATA_JSON_PATH).isNull();
     }
+
```

Comment:
(TODO - need to research why it fails) - "test_getItems_normal"

Reply:
@rliu-parasoft You can take a look at how Alfie reset the database for specific tests in this pull request.
https://github.com/parasoft/parasoft-demo-app/pull/119/files#diff-645b641a7060108ad2ed9c095776770a804bf16056cb20b0464be18dc4cb9472